vortex-api/src/main/java/mil/army/usace/hec/vortex/temporal/StaticIntervalTree.java

```diff
@@ -0,0 +1,113 @@
+package mil.army.usace.hec.vortex.temporal;
+
+import java.util.*;
+
+final class StaticIntervalTree<T extends Interval> {
```

Comment:
What is meant by "tree"?

Reply:
It's a type of data structure -- [example](https://www.geeksforgeeks.org/introduction-to-tree-data-structure/)